metpy/calc/tests/test_kinematics.py

```diff
@@ -654,3 +655,52 @@ def test_absolute_vorticity_asym():
     vort = absolute_vorticity(u.T, v.T, 1 * units.meters, 2 * units.meters,
                               lats.T, dim_order='xy')
     assert_almost_equal(vort, true_vort.T, 5)
+
+
+def test_potential_vorticity_unity_axis0():
+    """Test potential vorticity calculation with unity stability and height on axis 0."""
+    u = np.array([[100, 90, 80, 70],
```

Comment:
Depending on what we decide about this type of test that results in all zeros if it passes, I'll probably pull all of this data out in a fixture and/or parameterize some. Again, I like these tests because looking at the formula it's really clear what the output should be, but they are more expensive to run.

Reply:
This test is fine. Expensive is relative, and doing math on 16 values is not a huge burden.